ddtrace/internal/remoteconfig/client.py

```diff
@@ -227,13 +231,22 @@ def __init__(self):
             if container_id is not None:
                 self._headers["Datadog-Container-Id"] = container_id
 
+        tags = global_config.tags.copy()
+        if ddtrace.config.env:
+            tags["env"] = ddtrace.config.env
+        if ddtrace.config.version:
+            tags["version"] = ddtrace.config.version
+        tags["tracer_version"] = tracer_version
+        tags["host_name"] = get_hostname()
+
         self._client_tracer = dict(
             runtime_id=runtime.get_runtime_id(),
             language="python",
-            tracer_version=self._get_version(),
+            tracer_version=tracer_version,
             service=ddtrace.config.service,
             env=ddtrace.config.env,
             app_version=ddtrace.config.version,
+            tags=[":".join(_) for _ in tags.items()],
```

Comment:
can `:` exist in the tag value or key?

Reply:
Colons are allowed in tag values but I am not sure if it can exist in the key: [example](https://github.com/DataDog/dd-trace-py/blob/v1.8.0rc2/tests/tracer/test_utils.py#L469)

Was this edge case discussed in the remote config specification?

Reply:
The value of `ddtrace.config.tags` is the result of `parse_tags_str`, which can only handle colons in values, so we should be fine here.